test/src/main/java/org/apache/accumulo/test/VolumeChooserIT.java

```diff
@@ -132,7 +129,7 @@ public void configure(MiniAccumuloConfigImpl cfg, Configuration hadoopCoreSite)
 
     // Only add volumes 1, 2, and 4 to the list of instance volumes to have one volume that isn't in
     // the options list when they are choosing
-    cfg.setProperty(Property.INSTANCE_VOLUMES, v1 + "," + v2 + "," + v4);
+    cfg.setProperty(Property.INSTANCE_VOLUMES, v1 + "," + v2 + "," + v3);
```

Comment:
From a quick look, it seems this test trying to create a "hole" - does this invalidate the test if all volumes are present?

Reply:
The v3 variable was entirely unused in this test. I removed it, and renamed v4 to v3. That's it.